code/game/world.dm

```diff
@@ -134,6 +134,8 @@ GLOBAL_VAR(restart_counter)
 	GLOB.tgui_log = "[GLOB.log_directory]/tgui.log"
 	GLOB.world_shuttle_log = "[GLOB.log_directory]/shuttle.log"
 
+	GLOB.demo_log = "[GLOB.log_directory]/demo.log"

```

Comment:
This needs to be set to the startup log in /world/New()

Reply:
I already got this one and forgot to push

Reply:
Wait I just realized what you're saying here.
All that starting a log does is write round ID info to the top and a separator. That info (at the moment) breaks the viewer and prevents it from loading properly.